netbout-web/src/test/java/com/netbout/email/EmAliasTest.java

```diff
@@ -74,4 +75,34 @@ public void sendsEmail() throws Exception {
         );
     }
 
+    /**
+     * EmAlias doesn't send mail to the author.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void doesNotSendMailToAuthor() throws Exception {
+        final Postman postman = Mockito.mock(Postman.class);
+        final MkBase base = new MkBase();
+        final Alias author = new EmAlias(base.randomAlias(), postman);
+        final Alias friend = new EmAlias(base.randomAlias(), postman);
+        final Bout bout = author.inbox().bout(author.inbox().start());
+        bout.friends().invite(author.name());
+        bout.friends().invite(friend.name());
+        bout.messages().post("hello");
+        final ArgumentCaptor<Envelope> captor =
+            ArgumentCaptor.forClass(Envelope.class);
+        Mockito.verify(postman, Mockito.times(1)).send(captor.capture());
+        final Message msg = captor.getValue().unwrap();
+        MatcherAssert.assertThat(
+            msg.getAllRecipients().length,
+            Matchers.is(1)
+        );
+        final InternetAddress recipientAddress =
+            (InternetAddress) msg.getAllRecipients()[0];
+        MatcherAssert.assertThat(
+            recipientAddress.getAddress(),
+            Matchers.is(friend.email())
+        );
+    }
```

Comment:
@mbarbieri you can do one assertion, comparing the arguments list directly to a list of InternetAddresses containing only friend.email()...